be/src/cloud/cloud_internal_service.cpp

```diff
@@ -149,13 +151,26 @@ bvar::Adder<uint64_t> g_file_cache_warm_up_rowset_request_to_handle_slow_count(
         "file_cache_warm_up_rowset_request_to_handle_slow_count");
 bvar::Adder<uint64_t> g_file_cache_warm_up_rowset_handle_to_finish_slow_count(
         "file_cache_warm_up_rowset_handle_to_finish_slow_count");
+bvar::Adder<uint64_t> g_file_cache_warm_up_rowset_wait_for_compaction_num(
+        "file_cache_warm_up_rowset_wait_for_compaction_num");
+bvar::Adder<uint64_t> g_file_cache_warm_up_rowset_wait_for_compaction_timeout_num(
+        "file_cache_warm_up_rowset_wait_for_compaction_timeout_num");
 
 void CloudInternalServiceImpl::warm_up_rowset(google::protobuf::RpcController* controller
                                               [[maybe_unused]],
                                               const PWarmUpRowsetRequest* request,
                                               PWarmUpRowsetResponse* response,
                                               google::protobuf::Closure* done) {
     brpc::ClosureGuard closure_guard(done);
+    std::shared_ptr<bthread::CountdownEvent> wait = nullptr;
+    timespec due_time;
+    if (request->has_sync_wait_timeout_ms() && request->sync_wait_timeout_ms() > 0) {
```

Comment:
consider log every time that request with a wait timeout
compaction is much less than load, it is not verbose to log

Reply:
info: tablet id and timeout.
also consider logging at the stub side too

Reply:
there is the log at L376